src/lava/lib/dl/netx/hdf5.py

```diff
@@ -53,6 +53,12 @@ class Network(AbstractProcess):
         neuron's reset parameter. None means no reset. Defaults to None.
     reset_offset: int
         determines the phase shift of network reset if enabled. Defaults to 0.
+    sparsity_map : boolean array, optional
```

Comment:
Let's use `sparse_fc_layer` instead of `sparsity_map`

Reply:
One problem with calling it` sparse_fc_layer` I could see is that sparsity may be applied intentionally to other layer types besides fully connected layers in the future.  For example, it is conceivable to reap some benefit from sparse convolutional operations on the Loihi2. This all depends on your (and intel's broadly) roadmap of development for lava and loihi2 though.

Reply:
Sparse convolution might be handled differently. That's what I meant `sparse_fc_layer` just toggles the sparsity of dense layer for now. At a later point `sparse_conv_layer` can similarly be introduced.

Reply:
Okay it has been renamed. I also added some simple unit tests that check to see if the synapses are indeed sparse when `sparse_fc_layer`  or (in the case of blocks) `sparse_synapse` is set to True.